### DISCUSSION TITLE
Embeded docker code executions / Docker execution tool

### DISCUSSION THREAD

**Discussion by joaomdmoura** [crewAIInc/crewAI#1137] [Feature Requests]
> Hey João, can it exec code in a docker container like autogen? 
> I have a docker image with specific libraries installed. Autogen is close to perfect, working directory saves the code plus any input output files but it does not have sequential execution like crew AI.

<img width="590" alt="Screenshot 2024-01-10 at 11 25 12" src="https://github.com/joaomdmoura/crewAI/assets/667063/3a783121-df76-4a47-a312-f7edefecf38b">

**Comment by arnonel** [comment]
Any updates on docker ?

**Comment by joaomdmoura** [comment]
currently WIP, will be available in the next version
--
João Moura
@joaomdmoura <http://twitter.com/joaomdmoura>


Em qua., 6 de mar. de 2024 às 11:37, arnonel ***@***.***>
escreveu:

**Comment by yelloSA96** [comment]
Did you want some help @joaomdmoura? 
Have some experience, keen to aid the development on this project if I can.
Apologies if not posting in the right place, first time commenting outside of my repos.
